spec/resty/resolver_spec.lua

```diff
@@ -176,10 +195,25 @@ search localdomain.example.com local
 nameserver 127.0.0.2
 nameserver 127.0.0.1
 ]])
+       
```

Comment:
Bunch of whitespace here. We probably should have some linter for that.

Reply:
yes I'm trying to get this vscode right but I think I will just go back to VI

Reply:
done in 3c177e4b05d3eb9af3868f8ad9710f6aa193e5ce